src/reference-data/types.ts

```diff
@@ -1,9 +1,14 @@
 import {Polygon} from 'geojson';
 
-export type Language = 'nob' | 'nno' | 'nor' | 'eng';
+export enum Language {
```

Comment:
Should this be named something like `ReferenceDataLanguage` instead? To avoid importing wrong language

Reply:
Yeah. Also this type is only used in the utility method `getReferenceDataText`. I'll just define it in that utils file without export, so it doesn't show up as importable other places in the app.